package-lock.json

```diff
@@ -19142,48 +19142,22 @@
       "dev": true
     },
     "tipping-contract": {
-      "version": "github:aeternity/tipping-contract#e89ff0e85583f3b621c54daa72f37313f127cba1",
+      "version": "github:aeternity/tipping-contract#6b2029c06dc70737c1e09aaceae2a5623c4551a4",
```

Comment:
are you changing package-lock intentionally?

Reply:
I need to update (delete-reinstall) the package for the new v3 features. Although I'm not sure if this change in the package-lock is needed

Reply:
Consider to specify the commit explicilty in package.json, like:
```
"tipping-contract": "github:aeternity/tipping-contract#6b2029c06dc70737c1e09aaceae2a5623c4551a4",
```